packages/utilities/src/KeytipConstants.ts

```diff
@@ -0,0 +1,18 @@
+export const KTP_PREFIX = 'ktp';
+export const KTP_SEPARATOR = '-';
+export const KTP_FULL_PREFIX = KTP_PREFIX + KTP_SEPARATOR;
+export const DATAKTP_TARGET = 'data-ktp-target';
+export const DATAKTP_EXECUTE_TARGET = 'data-ktp-execute-target';
+export const KTP_LAYER_ID = 'ktp-layer-id';
+export const KTP_ARIA_SEPARATOR = ', ';
+export const KTP_ARIA_SEPARATOR_ID = 'ktp-aria-separator-id';
+
+// Events
+export namespace KeytipEvents {
+  export const KEYTIP_ADDED = 'keytipAdded';
```

Comment:
can you move the keytips stuff out of the utilities package? This will be included in the PLT1 bundle. It should only be in the keytips project in oufr.